ecosystem/web-wallet/src/utils/account.ts

```diff
@@ -26,3 +26,29 @@ export function createNewAccount (): Result<AptosAccount, Error> {
   // todo: make request to create account on chain
   return ok(account)
 }
+
+export function getLocalStorageState (): LocalStorageState | null {
+  if (typeof window === 'undefined') {
```

Comment:
When is window undefined?

Reply:
I don't think there is a time it is, was copy/paste from @hariria's UseWalletState - will delete unless you were running into a weird thing Andrew?